src/vivarium_census_prl_synth_pop/results_processing/formatter.py

```diff
@@ -31,24 +31,18 @@ def format_address_id(data: pd.DataFrame) -> pd.Series:
 
 def get_state_abbreviation(data: pd.DataFrame) -> pd.Series:
     state_id_map = {state: state_id for state_id, state in metadata.CENSUS_STATE_IDS.items()}
-    state_name_map = data["state"].map(state_id_map)
+    state_name_map = data["state_id"].map(state_id_map)
     return state_name_map.map(metadata.US_STATE_ABBRV_MAP)
 
 
-def get_state_id(data: pd.DataFrame) -> pd.Series:
-    return data["state"]
-
-
 # Fixme: Add formatting functions as necessary
```

Comment:
Remove fixme?

Reply:
Nah, we're gonna keep needing it. Maybe `Fixme` is more `NB` or `Nota bene`